pzblend/pzblend.py

```diff
@@ -20,6 +20,7 @@
 import gzip
 import joblib
 import random
+import skgof
```

Comment:
I think we should add `skgof` to `install_requires` in `setup.py` then.

Reply:
sounds good!